CLOUDFRONT.md

```diff
@@ -1,3 +1,37 @@
+## Implementing CloudFront CDN in-front of ObjectFS S3 Bucket
+The following steps outline how to create an S3 bucket for ObjectFS, configure moodle to use this, 
+and then how to implement the CloudFront CDN (Content Delivery Network) to securely sit infront of the 
+S3 Bucket, so that content delivery maybe off-loaded from the moodle servers to the CDN.  This will 
+typically result in faster access for users to content due do caching by the CDN, and less load on the
+moodle servers.
+
+The following steps implement the following high level objectives:
+1. Grant the Cloudfront Distribution access to the S3 bucket for ObjectFS
+ 
+  - existing steps in document are for "Legacy access identies"
+		"Use a CloufFront origin access identity (OAI) to access the S3 Bucket"
+
+  - use "Origin access control settings (recommended)" 
```

Comment:
This is a little bit confusing. Are you intending on adding the instructions for this new method?

Reply:
I have noted that text in quotes is from the AWS console screen.

There are is much required in terms of instructions, I have added notes under steps 4 & 5 of "Create CloudFront Distribution" where this option is implemented.